### ISSUE TITLE
Feature request: inline way of telling clonedigger to skip parts of code

### ISSUE THREAD

**Issue by pitchforks** [jlachowski/clonedigger#4]
There is the --func-prefixes option but it's not flexible enough. I'd rather want clonedigger to be also able to do what other code analysis tools (pylint, coverage) can - skip explicitly marked parts of code.

For example something like:

```
def foo():  # clonedigger: skip
```

would skip the function foo()

Or

```
class Bar:  # clonedigger: skip
```

would skip the whole class Bar

**Comment by floer32** [comment]
Yeah clonedigger needs something like this. But it would be even better if it were regexes or something that didn't clog up your code. Because I'm seeing now that it gets false positives for _docstrings_ and _doctests_! I can't figure out how to fix it in the source code just yet but I'll keep trying...
